src/worker.py

```
@@ -1296,6 +1315,111 @@ async def _dispatch(request, env):
+            "SELECT id FROM comments WHERE id = ? AND activity_id = ?"
+        ).bind(parent_id, activity_id).first()
+        if not parent:
+            return err("Parent comment not found", 404)
+
+    cid = new_id()
+    try:
+        await env.DB.prepare(
+            "INSERT INTO comments (id, activity_id, user_id, body, parent_id) "
+            "VALUES (?, ?, ?, ?, ?)"
+        ).bind(cid, activity_id, user["id"], encrypt(text, enc_key), parent_id).run()
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)